### PR TITLE
Do not generate dashboard groups for versioned projects in testgrid

### PR DESCRIPTION
**Pull request by Fredy-Z**
Testgrid does not allow dashboard groups having the same name as other dashboards. So skip versioned projects when we create dashboard groups, since each of them has only one dashboard.

### CODE REVIEW

**Comment by adrcunha** [comment]
This is just removing the offending groups, without actually fixing the problem. Please rollback the config and fix the dashboard grouping.

File: ci/prow/make_config.go [hunk 1]
```diff
@@ -1485,15 +1485,16 @@ func generateDashboardsForReleases() {
...
-			repos := metaData[projName]
-			for _, repoName := range repoNames {
-				if _, exists := repos[repoName]; exists {
-					dashboardRepoNames = append(dashboardRepoNames, buildProjRepoStr(projName, repoName))
-				}
+			continue
+		}
+
+		dashboardRepoNames := make([]string, 0)
+		repos := metaData[projName]
```
Comment (srinivashegde86) [inline ci/prow/make_config.go]: additional var not needed.

Comment (Fredy-Z) [reply ci/prow/make_config.go]: Hmm, I think having this var can make the code a bit clearer.

**Pull request merged**